### ISSUE TITLE
Display TG name instead number on mian window

### ISSUE THREAD

**Issue by sp2ong** [ea3ihi/DMRStation#4]
It will nice to have the display on Mina window TG  Name instead TG Number if the name of talk group exist in file tgs.dat

**Comment by ea3ihi** [comment]
It seems like the fixed layout widget does not support background-color,
here is the result of applying this css:

* {
background-color: #FF00FF;
}

[image: image.png]

**Comment by sp2ong** [comment]
David 

If we add to 
<object class="GtkFixed" id="fixed3">

the name of widget like:

<property name="name">mainW</property>

I suppose that we need add in main.c

GtkWidget       *mainW;

and

mainW = GTK_WIDGET(gtk_builder_get_object(builder, "mainW"));

to use #mainw in styles.css to set background color ?

In GTK3 exist function gtk_widget_override_background_color

**Comment by ea3ihi** [comment]
No, that is not needed at all. If you use glade you can see how styles are applied to the widgets

**Comment by sp2ong** [comment]
Ok, the main widow used GTKFixed and I found the information:

Some widgets don't have their own Gdk::Window. They draw on their parent container's window. Some of those widgets don't draw their background. E.g. Gtk::Label does not draw its background. Probably Gtk::Fixed is also one of the widgets that don't draw their background. There are (at least) two solutions:
1. Change the background of Gtk::Fixed's parent container.
If that's not suitable, perhaps too much background is changed, then
2. Put the Gtk::Fixed in a Gtk::EventBox, and change the background color of the Gtk::EventBox.

**Comment by ea3ihi** [comment]
Look at the screen capture I sent, where the background color was applied to all objects, but it did bolt affect the fixed item. At least in glade, I did not compile and run the app

**Comment by ea3ihi** [comment]
Labelled as enhancement
